Containerfile

```diff
@@ -7,9 +7,29 @@ USER root
 RUN apt update -y && apt install -y libgdal-dev libproj-dev libmpfr-dev && apt clean
 
 # PrevMap requires Terra and raster, which fail to install inside R. So install with mamba instead:
-RUN mamba install r-glmnet r-here r-igraph r-keras r-raster r-rmpfr r-ggraph r-tensorflow r-terra r-tidygraph
-
-RUN R -e "install.packages(c('assist', 'arrow', 'date', 'expm', 'filling', 'geosphere', 'imputeTS', 'PrevMap', 'tictoc', 'softImpute'), repos = 'https://cloud.r-project.org/', Ncpus = parallel::detectCores())"
```

Comment:
I noticed that `PrevMap` isn't in the mamba list.  Is this no longer needed?

Reply:
It's not in the conda repo. It's still in line 32 the "Old" way.